lua/entities/sent_tardis/init.lua

```diff
@@ -2003,7 +2003,10 @@ local E2Commands = {
 }
 
 function ENT:HandleE2(cmd, ...)
-	if not E2Commands[cmd] then return end
+	if not E2Commands[cmd] then 
+		error("TARDIS E2 Handler for command '"..cmd.."' is not registered")
```

Comment:
This should probably have the ,0 thing as well for the E2 errors

Reply:
I was going to counter that there's no way to manually call commands, but since we might implement things on the Rewrite that legacy won't have, it'll seem like our fault.